### DISCUSSION TITLE
about prisma client py schema.prisma

### DISCUSSION THREAD

**Discussion by mash180sx** [RobertCraigie/prisma-client-py#1019] [Q&A] [answered]
In prisma-client-python, can the schema.prisma file be moved to any folder of your choice? Also, is the multi-file Prisma schema feature supported with prismaSchemaFolder?

**Comment by RobertCraigie** [comment]
> In prisma-client-python, can the schema.prisma file be moved to any folder of your choice?

yes, you can tell prisma to use a different schema with `--schema=/path/to/schema.prisma`

> Also, is the multi-file Prisma schema feature supported with prismaSchemaFolder?

yes :)

**Reply by mash180sx** [reply]
> > In prisma-client-python, can the schema.prisma file be moved to any folder of your choice?
> yes, you can tell prisma to use a different schema with --schema=/path/to/schema.prisma

Can I write some config file, like pyproject.toml?

**Reply by RobertCraigie** [reply, marked as answer]
Unfortunately that is not supported

**Reply by mash180sx** [reply]
Thanks, your reply!
